hardhat.config.ts

```diff
@@ -83,6 +85,11 @@ const config: HardhatUserConfig = {
     optimismkovan: getChainConfig("optimismkovan"),
     optimism: getChainConfig("optimism"),
     harmonytestnet0: getChainConfig("harmonytestnet0"),
+    neonlabs: {
```

Comment:
Can we please have the same pattern as other chains i.e. `getChainConfig` here as well.
Also, now that the repo utilises private key instead of mnemonic, it makes sense to call it private key and do the config accordingly.

Reply:
It would better to leave mnemonic as it is and add privateKey there.

I am not sure if other chains need private keys with `0x`, if it does then store it as `0x` or change the config accordingly.